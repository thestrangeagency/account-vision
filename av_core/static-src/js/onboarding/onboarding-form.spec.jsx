import React from 'react';
import { shallow } from 'enzyme';

import OnboardingForm from './onboarding-form';

test('it renders', () => {
  const wrapper = shallow((
    <OnboardingForm
      action={() => {}}
      nextPage={() => {}}
      userId=""
    />
  ));

  expect(wrapper).toMatchSnapshot();
});
