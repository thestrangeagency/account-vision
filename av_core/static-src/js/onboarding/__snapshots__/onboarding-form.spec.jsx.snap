// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
<StepForm
  generalError={null}
  isDisabled={false}
  onSubmit={[Function]}
  steps={Array []}
  validationError={null}
  wasValidated={false}
/>
`;
